todo_app/templates/index.html

```diff
@@ -11,8 +11,19 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.title }} -- {{ item.status }}</li>
+        <form id="completeItem{{item.id}}" method="POST" action="/complete/{{item.id}}">
+          <input type="submit" value="Complete Item" />
+          <a onclick="deleteItem({{ item.id }})" href="#">Delete</a>
```

Comment:
See my other comment about redirects from app.py. This should no longer have onclick, and should have `href="{{ url_for('deleteItem', id=item.id) }}"` instead

Also, this doesn't really belong inside the complete item `<form>`

Reply:
is it possible to call a http `DELETE` using this method? I couldn't find how, so I created the ajax method in javascript